src/dom_cfg/rpi_5.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2024 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <domain.h>
+#include <zephyr/xen/public/domctl.h>
+#include <string.h>
+
+/*
+ * Example Xen guest domains configuration file for "rpi_5" which contains
+ * - domd_cfg_0 as DomD example which uses real HW "/soc/gpio@7d517c00" and can be used to run
+ *   samples/basic/blinky sample.
+ * - domd_cfg_1 as DomU, no real HW. It can be used to run any Zephyr example not accesing HW.
+ *   For example samples/synchronization
+ */
+
+extern char __img_domu_start_1[];
+extern char __img_domu_end_1[];
+extern char __dtb_domu_start_1[];
+extern char __dtb_domu_end_1[];
+
+static int load_domd_image_bytes_1(uint8_t *buf, size_t bufsize,
```

Comment:
Naming again
Symbols are took from nowhere, function called domd, uses symbols for domu